### PR TITLE
REM-327

### PR DESCRIPTION
**Pull request by omarZrayka23**
i add create customedatepickerHijri component  and import in  individual client and cash invoice

### CODE REVIEW

File: src/pages/clients-list/forms/ClientTemplateForm.js [hunk 1]
```diff
@@ -697,6 +698,7 @@ const ClientTemplateForm = ({ recordId, labels, plantId, maxAccess, allowEdit =
       title: labels.beneficiaries
     })
   }
+  console.log(clientIndividualFormik)
```
Comment (hadichahine) [inline src/pages/clients-list/forms/ClientTemplateForm.js:701]: Remove

File: src/components/Inputs/CustomDatePickerHijri.js
Comment (hadichahine) [inline src/components/Inputs/CustomDatePickerHijri.js:1]: Let's have it take the same props as the normal datepicker, same behavior too.

**Review by hadichahine — CHANGES_REQUESTED** [review]

File: src/pages/currency-trading/forms/TransactionForm.js [hunk 2]
```diff
@@ -1024,13 +1025,28 @@ export default function TransactionForm({ recordId, labels, access, plantId }) {
                         label={labels.birth_date}
                         value={formik.values?.birth_date}
                         required={true}
-                        onChange={formik.setFieldValue}
+                        onChange={(name, value) => {
+                          formik.setFieldValue('birth_date', new Date(value)?.getTime() || '')
```
[the remark this answers is not in the document]
Comment (omarZrayka23) [reply src/pages/currency-trading/forms/TransactionForm.js:1029]: i  added new Date(value)?.getTime() to convert date to tmestamp cz hijri value read only timestamp

File: src/components/Inputs/CustomDatePickerHijri.js
Comment (omarZrayka23) [reply src/components/Inputs/CustomDatePickerHijri.js:1]: i do not need all the props why add props not needed

**Pull request merged**